File: The Core/56-html-end-tag-by-start-tag.js
// https://app.codesignal.com/arcade/code-arcade/book-market/MX94DWTrwQw2gLrTi

function solution(startTag) {
    // without fancy powerful regex

    const firstSpace = startTag.indexOf(' ');

    let tag;
    if (firstSpace !== -1) {
        tag = startTag.slice(1, firstSpace);
    } else {
        tag = startTag.slice(1, startTag.length - 1);            
    }
    
    return `</${tag}>`;
}

File: The Core/56-html-end-tag-by-start-tag.go
// https://app.codesignal.com/arcade/code-arcade/book-market/MX94DWTrwQw2gLrTi

func solution(startTag string) string {
    var firstSpaceIndex = strings.Index(startTag, " ")
    
    var tag string;
    
    if firstSpaceIndex != -1 {
        tag = "</" + startTag[1 : firstSpaceIndex] + ">"
    } else {
        tag = "</" + startTag[1 : len(startTag) - 1] + ">"        
    }
    
    return tag
}
